ios/Classes/NetIamyellowTiwsWSProxy.m

```diff
@@ -121,8 +124,9 @@ -(void)close:(id)args
 
 -(void)send:(id)msg
 {
-    ENSURE_SINGLE_ARG(msg, NSString);
-
+    // not sure why tianium is passing a __NSArrayM with inside the TiBLob
+    if (![msg isKindOfClass:[NSString class]])
+        msg = [msg[0] data];
```

Comment:
I see.. thanks for taking the time to explain it.
I have fixed my code accordingly. Do you think there could be a better way to handle the type check?